setup.py

```diff
@@ -24,7 +24,7 @@ def read(*filenames, **kwargs):
     description='Add django enhanced Generic Foreignkey.',
     # long_description=read('README.txt', 'CHANGES.txt'),
     keywords='django generic foreignkey',
-    url='https://github.com/BrnoPCmaniak/django-egfk',
+    url='https://github.com/monterail/django-egfk',
```

Comment:
Also, you can do this only with your fork. I changed my username to "fdobrovoly" at that time that's why there is something different. But do not push this upstream to here it does not make sense.